src/molecules/formfields/shared/formfields.module.scss

```diff
@@ -54,39 +47,7 @@ $focused-border-color: color('brand-3');
   }
 }
 
-.select {
-  @include typography-regular();
-  @include typography-6(false);
-
-  border: 0;
-  width: 100%;
-  padding: ru(.5) ru(.75);
-  position: relative;
-  background: transparent;
-  -webkit-appearance: none;
-
-  &:focus {
-    outline: none;
-  }
-
-  &:invalid {
-    color: color('neutral-4');
-  }
-
-  &::placeholder {
-    color: color('neutral-4');
-  }
-}
-
-// -- Additional Elements
-.tooltip-wrapper {
-  position: absolute;
-  top: ru(.75);
-  right: ru(.75);
-}
-
-
-/* -- State:focus  -- */
+/* --  state:focus  -- */
 .focused {
```

Comment:
@patrickkim You could nest the classes in the placeholder, I believe -- it's not much of a difference, but it will scope it to whatever uses the placeholder only, enforcing our convention.